moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -232,13 +232,13 @@ const ProjectActivityLog = () => {
                       >
                         <Box display="flex" p={0}>
                           <Box p={0}>
-                            <Avatar
-                              alt={getInitials(change?.moped_user)}
-                              src="/moped/static/images/avatars/userAvatar.jpg"
+                            <CDNAvatar
                               className={classes.avatarSmall}
-                            >
-                              {getInitials(change?.moped_user)}
-                            </Avatar>
+                              src={change?.moped_user?.picture}
+                              initials={getInitials(change?.moped_user)}
+                              userColor={null}
+                              useGenericAvatar={true}
+                            />
```

Comment:
Replace Avatar with our custom component, passing all the necessary fields and overrides.